projects/cashmere/src/lib/form-field/hc-form-field.component.html

```diff
@@ -1,5 +1,5 @@
-<div [ngClass]="inline ? 'hc-form-field-wrapper-inline' : 'hc-form-field-wrapper'">
-    <span [ngClass]="inline ? 'hc-form-field-label-wrapper-inline' : 'hc-form-field-label-wrapper'">
+<div [ngClass]="{'hc-form-field-wrapper-inline': inline, 'hc-form-field-wrapper': !inline, 'hc-form-field-no-label': !_labelChildren.length}">
```

Comment:
IMHO this line is really hard to read and reason with.  Perhaps if you broke it up a bit:
```html
<div [class.hc-form-field-wrapper-inline]="inline"
     [class.hc-form-field-wrapper]="!inline"
     [class.hc-form-field-no-label]="!_labelChildren.length">
```

Reply:
There are actually two other spots in the template that do something similar, so I'm going to change all three as you indicated.